### PR TITLE
supports log_loss in the python front-end

### PR DESCRIPTION
**Pull request by gavin1332**

### CODE REVIEW

**Review by panyx0718 — DISMISSED** [review]

**Comment by pkuyym** [comment]
Please add a test for the python wrapper, you can refer to 'https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/tests/unittests/test_layers.py#L362'

**Comment by pkuyym** [comment]
Please refine the doc, current comment didn't follow the format described here http://www.paddlepaddle.org/docs/develop/documentation/fluid/zh/dev/api_doc_std_cn.html

**Comment by pkuyym** [comment]
Please refine the doc, current comment didn't follow the format described here http://www.paddlepaddle.org/docs/develop/documentation/fluid/zh/dev/api_doc_std_cn.html

**Comment by panyx0718** [comment]
@gavin1332 need any help?

**Comment by gavin1332** [comment]
@panyx0718 I have reformated comments and add an unittest as @pkuyym suggested

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -975,6 +976,44 @@ def cross_entropy(input, label, soft_label=False):
...
+
+    This layer accepts input predictions and target label and returns the
+    negative log loss.
+
+    .. math::
+
+        Out = -label * log(input + epsilon) - (1 - label) * log(1 - input + epsilon)
+
+    Args:
+        input (Variable|list):  a 2-D tensor with shape [N x 1], where N is the batch size.
```
Comment (pkuyym) [inline python/paddle/fluid/layers/nn.py]: exceeded 80 columns

Comment (gavin1332) [reply python/paddle/fluid/layers/nn.py]: sorry... in my memory, the limitation of google style is 100. I will update soon

Comment (gavin1332) [reply python/paddle/fluid/layers/nn.py]: @pkuyym done

Comment (gavin1332) [reply python/paddle/fluid/layers/nn.py]: problem fixed

**Comment by gavin1332** [comment]
Could this pr be merged? @pkuyym

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -975,6 +976,46 @@ def cross_entropy(input, label, soft_label=False):
...
+    This layer accepts input predictions and target label and returns the
+    negative log loss.
+
+    .. math::
+
+        Out = -label * log(input + epsilon) 
+              - (1 - label) * log(1 - input + epsilon)
+
+    Args:
+        input (Variable|list):  a 2-D tensor with shape [N x 1], where N is the
```
Comment (pkuyym) [inline python/paddle/fluid/layers/nn.py:992]: emmm, seems exceeded 80 columns again.

**Review by pkuyym — DISMISSED** [review]
Thanks, LGTM

**Comment by gavin1332** [comment]
why can't i pass the TeamCity build, though i have already regitered

**Comment by gavin1332** [comment]
And I have signed the CLA, but it seems no effect.

**Comment by panyx0718** [comment]
Is the email you use to sign-in and the email you use to commit the same email?

**Comment by gavin1332** [comment]
Thanks for your advice. I passed the CLA assignment after adding my Baidu email account to my profile. However, I am still trapped by the TeamCity building. Should I commit again?

**Comment by panyx0718** [comment]
Right, you should probably update your commit to trigger the CI

**Pull request closed without merging**